src/main/java/org/checkerframework/checker/dependencyinjection/DependencyInjectionAnnotatedTypeFactory.java

```diff
@@ -102,6 +117,38 @@ private void printKnownBindings() {
     System.out.println();
   }
 
+  /** Debugging method that pretty prints {@code knownBindings} */
+  private void printDependencies() {
+    System.out.println("Dependencies:");
+    DependencyInjectionAnnotatedTypeFactory.injectionPoints.forEach(
```

Comment:
This is the same code used by `printKnownBindings`. It is possible to abstract this into a three methods:
* one with the core logic, which operates on an arbitrary map
* an implementation of this method that calls that method on `injectionPoints`, probably with a constant string parameter set to `"Dependencies:"`
* an implementation of `printKnownBindings` that similarly delegates